paddle/fluid/platform/dynload/dynamic_loader.cc

```diff
@@ -117,16 +121,15 @@ static inline void* GetDsoHandleFromSearchPath(const std::string& search_root,
     // search xxx.so from custom path
     dlPath = join(search_root, dso_name);
     dso_handle = dlopen(dlPath.c_str(), dynload_flags);
+#if !defined(_WIN32)
+    auto errorno = dlerror();
+#else
+    auto errorno = GetLastError();
+#endif  // !_WIN32
     // if not found, search from default path
     if (nullptr == dso_handle) {
       LOG(WARNING) << "Failed to find dynamic library: " << dlPath << " ("
-                   << dlerror() << ")";
-      if (dlPath.find("nccl") != std::string::npos) {
-        std::cout
-            << "You may need to install 'nccl2' from NVIDIA official website: "
```

Comment:
why we need to delete this part? This part is used to show user nccl2 not found error is not our fault

Reply:
repicked